README.CentOS7.md

```diff
@@ -103,6 +103,31 @@ Customize and Verify your settings:
  * DocumentRoot should point to `/var/www/loris/htdocs`
  * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
 
+Modify the Apache configuration file for our production environment.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
```

Comment:
this OS x tools isn't the same on Linux.

People can use vi (installed by default), nano (quite often install by default now), emacs, ...
Would be good to either 
  - tell user to use their preferred file editor or
  - give them instruction on how to navigate, change detail, save, ...

Plus, the http config file is `/etc/httpd/conf/httpd.conf`

Reply:
@PapillonMcGill thank you. Please see new commit for change.

Reply:
I don't think that target user will no how to use `vi`